WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -25,6 +25,11 @@ public class CoinJoinClient
 	private volatile bool _inCriticalCoinJoinState;
 	private static readonly Money MinimumOutputAmountSanity = Money.Coins(0.0001m); // ignore rounds with too big minimum denominations
 
+	// Maximum delay when spreading the requests in time, except input registration requests which
+	// timings only depends on the input-reg timeout.
+	// This is a maximum cap the delay can be smaller if the remaining time is less.
+	private TimeSpan MaximumRequestDelay { get; init; } = TimeSpan.FromSeconds(10);
```

Comment:
Do we need the `init`? This is in fact a constant so, i would go with the getter only or an `private static readonly`

Reply:
ACK